### DISCUSSION TITLE
为什么ant-design要忽略packge-lock.json和yarn.lock文件

### DISCUSSION THREAD

**Discussion by yiluxiangbei87110** [ant-design/ant-design#29446] [Q&A] [answered]
为什么ant-design要忽略packge-lock.json和yarn.lock文件？这样不会导依赖版本出现变更导致不必要的问题吗？

![image](https://user-images.githubusercontent.com/23131032/108942081-467f1080-7691-11eb-97f0-aea7c13b3c54.png)

**Comment by yoyo837** [comment, marked as answer]
antd 每个组件的功能大部分都从rc-* 包装而来，他们都遵循 [Semantic Versioning 2.0.0](http://semver.org/lang/zh-CN/) 语义化版本规范。忽略look文件，方便给rc-*快速发布path版本，而不需要频繁修改依赖的rc-*版本，测试用例也可以直接以最新patch运行，path 通常不会破坏功能，只有bugfix。

**Reply by yiluxiangbei87110** [reply]
thx
